site/cypress/e2e/Cypress-UI/sidebar.spec.js

```diff
@@ -11,9 +11,12 @@ function sidebarContains(title, extension, header = title) {
     cy.visit(extension);
 
     let selector = '#main > .content';
-    if (title === 'SQL Toolbox' || title === 'My Courses') {
+    if (title === 'SQL Toolbox') {
         selector = '#main > div[data-v-app] > .content';
     }
+    else if (title === 'My Courses') {
+        selector = '.home-content';
+    }
```

Comment:
This should really be done with a test selector instead of trying to match complex class-based selectors.   Doing so is outside the scope of this PR though.  Just noting it for future reference.